core/src/main/scala/ch/epfl/scala/index/newModel/NewProject.scala

```diff
@@ -63,6 +66,7 @@ object NewProject {
       Organization(org),
       repository = Repository(repo),
       githubInfo = githubInfo,
+      created = Some(Instant.now),
```

Comment:
I would prefer to say `None` that `Instant.now` which is not right.

Reply:
ok, I thought it was only used in tests